docs/releases.md

```diff
@@ -30,14 +30,10 @@ To generate the next release binary:
     - each release includes the sub headings "What's New", "What's Fixed", and "Known Issues"
     - consult with Product Management on wording if needed
 2. Commit and ultimately merge to `master` branch
-3. Merge the `master` branch into `production` branch and push to GitHub:
-    - `git checkout master`
-    - `git pull upstream master` (to make sure you have the latest)
-    - `git checkout production`
-    - `git pull upstream production` (to make sure you have the latest)
-    - `git merge master`
-    - `git push upstream production`
-    - Test Pilot's Jenkins will now build and sign the extension (see ["Extension Signing"](#extension-signing))
+3. Create a pull request comparing the `master` branch and `production` branch on GitHub
+    - Open the pull request so we can show the changes, CI status, and approvals
+    - Once the release has been reviewed, tested, and approved to go live, merge and close the pull request
```

Comment:
should this be a bit more specific on **who** reviews/tests/approves?  Tests is fairly obviously PI.  But should UX and/or Product have a go over on this, too?

Reply:
>should this be a bit more specific on who reviews/tests/approves? 

I think a little more specificity. In the spirit of adding true "instructions" I said we'd expect PI and Product reps to approve, so we should explicitly request them on the PR. ☑️